CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Bidirectional m:n association Subscribers/SubscriberLists (#254)
 
 ### Changed
+- Change composer configuration to reflect new repository name 'core'
```

Comment:
I propose this text instead:

Rename the Composer package to "phplist/core"

Please also add the PR number.